src/plugins/plugin-player.ts

```diff
@@ -33,9 +37,11 @@ interface PluginPlayerOpts {
  *
  * @param {function} initPlayerState - Function of type (playerID) => playerState.
  */
-const PlayerPlugin = ({ setup }: PluginPlayerOpts = {}): Plugin<
-  PlayerAPI,
-  PlayerData
+const PlayerPlugin = <PlayerState extends any = any>({
+  setup,
+}: PluginPlayerOpts = {}): Plugin<
```

Comment:
Shouldn’t this be `PluginPlayerOpts<PlayerState>` here?

Reply:
Yes, good catch, missed that